velox/exec/DistinctAggregations.cpp

```diff
@@ -123,10 +123,12 @@ class TypedDistinctAggregations : public DistinctAggregations {
         }
 
         rows.resize(data->size());
-        std::vector<VectorPtr> inputForAggregation_ =
-            makeInputForAggregation(data);
-        aggregate.function->addSingleGroupRawInput(
-            group, rows, inputForAggregation_, false);
+        if (rows.hasSelections()) {
+          std::vector<VectorPtr> inputForAggregation_ =
```

Comment:
Not related to the PR, but can we rename local var "inputForAggregation_" to "inputForAggregation"?